### PR TITLE
Adds in the Culinary Implant

### PR DESCRIPTION
**Pull request by Mitchs98**
**What does this PR do:**
Adds in a Culinary Implant. Basically like any other implant. A Quality of Life improvement for the chef, frees up pack space, keeps everything nice and organized in one button. Comes with a knife, rolling pin, Sushi Mat, and cookie cutter. Can be emagged to receive a Butcher's Cleaver(Not the antag Meat Cleaver). Requires the same research levels as the surgical and toolset implants.

**Changelog:**
:cl:
add: Adds in the Culinary Implant, researchable and printable at science. Same tech levels as Surgical and Toolset Implants.
/:cl:

### CODE REVIEW

**Comment by Kyep** [comment]
How strong is the butcher's cleaver? I envision lots of chefs using the cleaver from their implant as a weapon in permabrig, and security treating them worse due to this.

**Comment by lass9030** [comment]
Not enough knife. Needs more knife.
Looks good, though.

**Comment by Mitchs98** [comment]
> How strong is the butcher's cleaver? I envision lots of chefs using the cleaver from their implant as a weapon in permabrig, and security treating them worse due to this.

Fifteen Force. No stronger than the surgical implants' saw.

**Comment by taukausanake** [comment]
Needs a fork and spoon too just because. Also would be neat if you could have a salt and pepper shakers but need to refill them yourself. That might be a little dumb though.

I think a tray that you can't drop might be nice. If you put it away everything on it falls off. Just a food inventory for moving your stuff around

File: code/game/objects/items/weapons/kitchen.dm [hunk 1]
```diff
@@ -270,7 +282,9 @@
 	w_class = WEIGHT_CLASS_SMALL
 	attack_verb = list("rolled", "cracked", "battered", "thrashed")
 
-
+/obj/item/kitchen/sushimat/augment
+	desc = "A small wooden make to make sushi with."
```
Comment (farie82) [inline code/game/objects/items/weapons/kitchen.dm]: wooden make? probably a typo here :D

File: code/modules/surgery/organs/augments_arms.dm [hunk 2]
```diff
@@ -277,6 +277,24 @@
...
 
+/obj/item/organ/internal/cyberimp/arm/chef
+	name = "culinary implant"
+	desc = "A set of culinary tools hidden behind a concealed panel on the user's arm"
+	contents = newlist(/obj/item/kitchen/knife/augment, /obj/item/kitchen/rollingpin/augment, /obj/item/kitchen/sushimat/augment, /obj/item/kitchen/cutter/augment)
+	origin_tech = "materials=3;engineering=3;biotech=3;programming=2;magnets=3"
+	action_icon = list(/datum/action/item_action/organ_action/toggle = 'icons/obj/clothing/hats.dmi')
+	action_icon_state = list(/datum/action/item_action/organ_action/toggle = "chef")
+
+/obj/item/organ/internal/cyberimp/arm/toolset/l
```
Comment (Fox-McCloud) [inline code/modules/surgery/organs/augments_arms.dm]: Copy-paste fail.

**Comment by Fox-McCloud** [comment]
Don't really understand why there's subtypes for this augment.

They don't do anything special (nor do they need to, really).

File: code/modules/surgery/organs/augments_arms.dm (hunk 2 above)
Comment (Mitchs98) [reply code/modules/surgery/organs/augments_arms.dm]: Fuck. That's not as bad as when I made everything into knives...x.x

File: code/game/objects/items/weapons/kitchen.dm (hunk 1 above)
Comment (Mitchs98) [reply code/game/objects/items/weapons/kitchen.dm]: Indeed...Indeed it is.

**Comment by Mitchs98** [comment]
Oh? I thought that was needed to prevent fuckery/dropping it/what have you. I definitely can remove them if it's not, would of saved me quite a bit of trouble.

Same logic as a toolset implant, really. It's just quality of life. I'd personally use it, quite a few people said they would when I was going to make it a while back. I genuinely think it'd see more use than you think it would.

**Comment by davidchan13** [comment]
It's niche and while I don't see many of them being handed out, if feel that's more to having to get RnD/Robo to print one and then Robo/Medical to install it.

Would love to see one of those for Botany.

**Comment by Mitchs98** [comment]
> It's niche and while I don't see many of them being handed out, if feel that's more to having to get RnD/Robo to print one and then Robo/Medical to install it.
> 
> Would love to see one of those for Botany.

Botany implant is planned. I'm avoiding doing both at once due to merge conflicts ^^

**Comment by davidchan13** [comment]
I feel there is an argument to be made to just slap the botany and culinary tools into one implant though I know several people would argue that's too strong even when there is already precedent of chef growing their own food and botanists cooking meals. Though if there is one dream feature for a culinary toolset it'd be some manner of portable grinder for making flour/rice on the go but I don't think there is any easy way of doing that.

**Comment by Mitchs98** [comment]
I considered making them one implant, but as you said didn't want people to think it was too 'strong' or whatever. So it'll be a separate implant PR.

**Comment by taukausanake** [comment]
These implants go into the arm, don't they? Can't you just make them both and put them into separate arms? I'd rather they be separate personally as they are different jobs

**Comment by Fox-McCloud** [comment]
Botany has 2 items that are viable for an arm implant; their plant analyzer and the cultivator.

...They don't need one.

Allowing them to have plant bags, pest spray, syringes, or the likes that are retractable is a big no no; being able to selectively hide stuff like spray bottles and syringes is very likely going to be problematic.

**Comment by Pennwick** [comment]
> 
> 
> > How strong is the butcher's cleaver? I envision lots of chefs using the cleaver from their implant as a weapon in permabrig, and security treating them worse due to this.
> 
> Fifteen Force. No stronger than the surgical implants' saw. Chef also does not start with it, it must be implanted.

Actually the Surgical implant saw only has a force of 15. Its dulled specifically to lower its damage. Here are the current stats as I understand them.
Toolset: 15 burn welding tool, 20 damage cyborg knife if emagged.
Surgery: 10 damage scalpel or saw, none if emagged
Culinary: 10 damage knife, 15 damage butcher knife if emagged

**Comment by SkeletalElite** [comment]
Cyborg knife is 30 damage @Pennwick

**Comment by Pennwick** [comment]
@SkeletalElite 

I'm pretty sure its 20. Emag act adds a /obj/item/kitchen/knife/combat/cyborg knife to the set if it doesn't have one. And thats just a subtype of the combat knife, which is force 20.

**Comment by MrMagolor** [comment]
All we need now is a Service ERT.

**Comment by TDSSS** [comment]
>All we need now is a Service ERT.

Mop implant for general crew use when?

**Comment by Mitchs98** [comment]
> > > How strong is the butcher's cleaver? I envision lots of chefs using the cleaver from their implant as a weapon in permabrig, and security treating them worse due to this.
> > 
> > 
> > Fifteen Force. No stronger than the surgical implants' saw. Chef also does not start with it, it must be implanted.
> 
> Actually the Surgical implant saw only has a force of 10. Its dulled specifically to lower its damage. Here are the current stats as I understand them.
> Toolset: 15 burn welding tool, 20 damage cyborg knife if emagged.
> Surgery: 10 damage scalpel saw or drill, none if emagged
> Culinary: 10 damage knife, 15 damage butcher knife if emagged However the chef also has the easiest access to sharpening stones.

Huh. Didn't notice that. Either way. It can only be sharpened once, and it'd be no more powerful than the toolset implant. If you're trying to get implants for special emagged items.

**Comment by SkeletalElite** [comment]
I've used an emagged tool arm knife on a monkey and health analyzed it before. 100% does 30 brute

**Comment by UmeFuu** [comment]
I love this. Please yes.

**Comment by Fox-McCloud** [comment]
I just don't really think this is necessary, to be honest, it's so incredibly niche for a job that's also pretty niche and doesn't use tools nearly as often as surgeon or toolset user would.

**Comment by Mitchs98** [comment]
> I just don't really think this is necessary, to be honest, it's so incredibly niche for a job that's also pretty niche and doesn't use tools nearly as often as surgeon or toolset user would.

Most chefs use the tools I included in it pretty often actually. At-least...any good chef does for cooking.

**Comment by KasparoVy** [comment]
Deep frier and a pair of robist fusts

**Comment by Mitchs98** [comment]
Just poking this as it's been a fair bit. What's the status on acceptance/denial?

**Comment by Pennwick** [comment]
I've used an emagged tool arm knife on a monkey and health analyzed it before. 100% does 30 brute

That's proof of it being 20. Monkeys take 50% extra damage.

**Comment by variableundefined** [comment]
I'll have to agree with Fox here that the culinary implant is a bit too niche, so we'll be closing this PR.

**Pull request closed without merging**